contentreview-impl/impl/src/java/org/sakaiproject/turnitin/util/TurnitinLTIUtil.java

```diff
@@ -124,8 +124,13 @@ public int makeLTIcall(int type, String urlParam, Map<String, String> ltiProps){
 				return -3;
 			} else if(statusCode == 200){
 				log.debug("Status 200: OK request: " + defUrl);
-				log.debug("LTI props " + ltiProps.toString());
-				log.debug(method.getResponseBodyAsString());
+				if(method.getResponseBodyAsString().contains("Sorry, we could not process your request")){//TODO only way unless TII returns xml response
```

Comment:
Asking Jessica about this, I have always seen those messages in English but who knows. Anyway I really hate to check it that way.